src/c++/vdo/base/repair.c

```diff
@@ -51,6 +67,10 @@ struct recovery_point {
 	bool increment_applied;
 };
 
+#ifdef VDO_USE_ALTERNATE
+DEFINE_MIN_HEAP(struct numbered_block_mapping, replay_heap);
+#endif
+
 struct repair_completion {
```

Comment:
One last nit. Switch lines 72 and 73, so that if VDO_USE_ALTERNATE is false, there won't be two blank lines in a row.

Reply:
This one could use a blank line inside the #ifdef, but it's not worth bothering about.